### DISCUSSION TITLE
Indicate replaced objects

### DISCUSSION THREAD

**Discussion by Richard-L** [OpenRCT2/OpenRCT2#17906] [Feature requests]
The process of replacing certain objects with updated versions upon opening a park is quite clever. But since the feature isn't optional, and since there is no way of knowing for the majority of users an object was replaced by OpenRCT2, is there any chance you would consider adding a way to indicate replacements were undertaken upon load?

I was thinking perhaps in the scenery menu or object selection - tried some options:

![image](https://user-images.githubusercontent.com/4934209/187032081-bdfc9244-a58f-42e1-ad0a-74c155106aaa.png)

^perhaps on the right where I put the red circle, there could be an icon that toggles this info, and that's "off" by default.

Alternatively perhaps could a console command be added that lists all legacy and replaced object pairs, and that can be copy/pasted to the clipboard (or even just directly output to txt)?

Many thanks for considering.

**Comment by Derpercoaster** [comment]
I think doing this would require restructuring the game's internal logic. In other words, it would take a long time, and would be a lot of work. This is possible, but if anyone wants to implement it, it would have to be done during a very big release- likely on par with the .park release.

Also, clever idea to use the 'i' icon for this. It's an underused graphic, in my opinion.
